designs/credentials/credentials-management.md

```diff
@@ -0,0 +1,65 @@
+# Credentials Management
+
+This outlines how the Toolkit produces and obtains credentials.
```

Comment:
nit - "how the Toolkit produces and obtains credentials." could be part of the title